ZukoAzula2016/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//src/org/team1540/zukoazula/RobotTemplate.java=UTF-8
```

Comment:
@AidanSmith I'm fairly sure that I'm right here. When he loaded the file in eclipse, it loaded it using his system default utf-16. This broke the emoji. Then he changed the encoding to utf-8, which would of fixed the problem, if eclipse had not already rewritten the file. So I'm pretty sure this is an important setting to preserve.